numba/typedobjectutils.py

```diff
@@ -39,18 +40,34 @@ def _sentry_safe_cast(fromty, toty):
     """Check and raise TypingError if *fromty* cannot be safely cast to *toty*
     """
     tyctxt = cpu_target.typing_context
+    fromty, toty = map(types.unliteral, (fromty, toty))
     by = tyctxt.can_convert(fromty, toty)
+
+    def warn():
+        m = 'unsafe cast from {} to {}. Precision maybe lost.'
```

Comment:
```suggestion
        m = 'unsafe cast from {} to {}. Precision may be lost.'
```

Reply:
I also wonder if this should not refer to precision as down casting `int`'s may end up with overflow etc.